TEST-4/js/script.js

```diff
@@ -0,0 +1,28 @@
+const containerOne = document.getElementById("first-container");
```

Comment:
You have class `container-one` and id `first-container`, so it can be messy when your constant name is rely to the different class name. Better replace `containerOne` with `firstContainer`